DependencyInjection/SonataMediaExtension.php

```diff
@@ -111,6 +111,15 @@ public function load(array $configs, ContainerBuilder $container)
 
         if (isset($bundles['SonataAdminBundle'])) {
             $loader->load(sprintf('%s_admin.xml', $config['db_driver']));
+
+            if ($container->hasParameter('sonata.admin.configuration.security.role_admin')
+                && $container->hasParameter('sonata.admin.configuration.security.role_super_admin')) {
+                $container->getDefinition('sonata.media.security.superadmin_strategy')
+                    ->replaceArgument(2, array(
+                        $container->getParameter('sonata.admin.configuration.security.role_admin'),
+                        $container->getParameter('sonata.admin.configuration.security.role_super_admin')
```

Comment:
Add a `,` here at the end of the line to make StyleCI pass.

Reply:
Beat me to it :)

Reply:
Hey @jlamur thanks! I usually put a comma alter the last element... didn't know that StyleCI complains about that...